js/ftx.js

```diff
@@ -2065,11 +2065,11 @@ module.exports = class ftx extends Exchange {
         const defaultMethod = this.safeString (options, 'method', 'privateGetOrders');
         let method = this.safeString (params, 'method', defaultMethod);
         const type = this.safeValue (params, 'type');
-        const stop = this.safeValue (params, 'stop');
-        if (stop || (type === 'stop') || (type === 'trailingStop') || (type === 'takeProfit')) {
+        const isTriggerOrder = this.safeValue2 (params, 'trigger', 'stop');
```

Comment:
@ttodua We're missing some methods here (cancelOrder, cancelAllOrders, ...) can you apply the same logic there as well?

To avoid duplication, I'd suggest creating here a helper method like `isTriggerOrder` that would do all of those checks and omit it from params (similar to handleMarketTypeAndParams). By doing that, if we need to change it in the future, we only have to change it in one place, wdyt?

Reply:
Yeah, all your notes are good !

Reply:
Carlos, it turned out it needed some more updates and deeper tricks (with that strange ftx api), however, i consider that these are the correct changes and needed to make it work. 

side-note: they have this custom endpoint - https://docs.ftx.com/#get-trigger-order-triggers where you send the conditional-order's id and from where you get the regular order-id, which can be used with standard means. So, i've used that approach too to get the `order-id` (of the 'triggered' conditonal order, otherwise 'non-triggered' conditional orders doesn't provide any info from that endpoint), and then use that `order-id` to get the actual order's details.